notification/src/handler/notification/notification.handler.js

```diff
@@ -6,6 +6,25 @@ const { getNotificationForTracking } = require('../../utils/commons')
 const ctp = require('../../utils/ctp')
 const mainLogger = require('../../utils/logger').getLogger()
 
+class CommercetoolsError extends Error {
+  constructor({ stack, message, statusCode }) {
+    super()
+    this.stack = stack
+    this.message = message
+
+    /*
+      We do not want to block notifications coming by Adyen.
```

Comment:
We do not want but we do that if necessary. Why not just include the info:
```
recoverable: notification delivery can be retried by Adyen
non recoverable: notification delivery can not be retried by Adyen as it most probably would fail again

If commercetools status code is defined and is 5xx then return `500` to Adyen -> recoverable
If during communication with commercetools we got a `NetworkError` then return `500` -> recoverable
If commercetools status code is not OK but also not 5xx or 409 then return `accepted` -> non recoverable
```

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/629/commits/67b2baa4eee6587bf075d3729cf0dc03a94e81cf